lmos-runtime-bom/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Deutsche Telekom AG and others
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id("java-platform")
+    id("maven-publish")
+}
+
+javaPlatform {
+    allowDependencies()
+}
+
+dependencies {
+    val springBootVersion = "3.5.5"
+    val ktorVersion = "3.3.0"
+    val kotlinxVersion = "1.9.0"
+    val lmosRouterVersion = "0.9.0"
```

Comment:
This is a downgrade of the router version. 

Yesterday, some library version changes were merged by Dependabot. Please check these versions as well.